rerun-failed-run/rerun-failed-run.yml

```diff
@@ -0,0 +1,73 @@
+name: Retrigger failed run action
+
+on:
+  workflow_dispatch:
+    inputs:
+      run_id:
+        description: The id of the run which we want to retrigger
+        required: true
+      error_message:
```

Comment:
what if a team wants to have multiple error message strings to match?

Reply:
What about targeting this maybe in a follow up ticket as we would have to provide a way for them to see whether encountering all errors or at least one of the errors should retry the workflow?

Reply:
sure, sounds good. Just want to make sure the current architecture is not prohibiting that extension in the future.